### PR TITLE
tests: start using remote tools from snapd-testing-tools project in nested tests

### PR DESCRIPTION
**Pull request by sergiocazzolato**
This change introduces the new tools for remote commands from the
snapd-testing-tools project

This change starts using the remote.exec which replaces the tests.nested
exec and the remote.push by the tests.nested copy

The new remote commands have the same implementation then the
tests.nested but those include tests and are being used in other
projects successfully.

### CODE REVIEW

File: tests/nested/manual/remodel-simple/task.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 summary: verify a simple UC20 remodel
 details: |
   Execute a simple remodel scenario under the test-snapd brand. Both models use
-  the same test-snapd-whitebox device view store. The new model requires
-  additional snaps that are present in the test-snapd-whitebox store.
+  the same test-snapd-from device view store. The new model requires
```
Comment (mardy) [inline tests/nested/manual/remodel-simple/task.yaml:4]: This changes are unrelated.

**Review by mardy — COMMENTED** [review]
LGTM, but I'd move the inclusive wording changes to another PR.

**Review by Meulengracht — APPROVED** [review]
Looks nice, but I agree with the comments mardy provided

File: tests/nested/manual/remodel-simple/task.yaml (hunk 1 above)
Comment (sergiocazzolato) [reply tests/nested/manual/remodel-simple/task.yaml:4]: I see it is intentional to just check with woke the inclusive naming in the changed files.

This is the list of findings in case all the project files are analyzed
https://paste.ubuntu.com/p/9CtxVWgNgP/

Comment (mardy) [reply tests/nested/manual/remodel-simple/task.yaml:4]: OK, but the inclusive naming check is not blocking, so we don't necessarily have to fix the things it reports.

**Review by mardy — APPROVED** [review]
Code wise LGTM, I'd split the inclusive naming to another PR (or just ignore it), but it's a minor issue, so up to you :-)
